consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -835,6 +845,7 @@ impl<
                 ConsensusValue::MintTx(mint_tx) => {
                     mint_txs.push(mint_tx);
                 }
+                ConsensusValue::TimeStamp(timestamp) => timestamps.push(timestamp),
```

Comment:
Do you think we can enforce that there is only a single externalized timestamp? If not, then when/why would we see multiple?

Reply:
This is taken care of in #3733 now during block externalizing